component/azstorage/config.go

```diff
@@ -182,10 +183,12 @@ type AzStorageOptions struct {
 	MaxResultsForList       int32  `config:"max-results-for-list" yaml:"max-results-for-list"`
 	DisableCompression      bool   `config:"disable-compression" yaml:"disable-compression"`
 	Telemetry               string `config:"telemetry" yaml:"telemetry"`
-	HonourACL               bool   `config:"honour-acl" yaml:"honour-acl"`
-	CPKEnabled              bool   `config:"cpk-enabled" yaml:"cpk-enabled"`
-	CPKEncryptionKey        string `config:"cpk-encryption-key" yaml:"cpk-encryption-key"`
-	CPKEncryptionKeySha256  string `config:"cpk-encryption-key-sha256" yaml:"cpk-encryption-key-sha256"`
+	BlobFilter              string `config:"blobFilter" yaml:"blobFilter"`
```

Comment:
Move this new config towards the end of the structure so that diff does not show existing items.